curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/6625d84e92201f802eac3973.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 6625d84e92201f802eac3973
+title: Task 46
+challengeType: 22
+dashedName: task-46
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: When the documentation is extensive, start with the introductory sections.
+-->
+
+# --description–-
```

Comment:
```suggestion
# --description--
```

One of the dashes wasn't a regular dash or something - so the description wasn't showing up.